test/parallel/test-http-response-no-headers.js

```diff
@@ -50,8 +49,6 @@ function test(httpVersion, callback) {
   });
 }
 
-test('0.9', function() {
```

Comment:
The 0.9 test was removed because HTTP/0.9 responses do not have response lines or headers, just the entity body, so you should never see a response like `HTTP/0.9 200 OK`